aazztech/directorist
# Add/sort by distance

## PR Type
What kind of change does this PR introduce?


## Description
How to reproduce the issue or how to test the changes

1. Enable Distance option from Settings > All Listings/Search Result > Sort By Dropdown
2. Do a radius search
3. Check the Sort By option to see if it sorts correctly

## Any linked issues
Fixes #

## Checklist
